acceptance/acceptance_test.go

```diff
@@ -716,14 +712,12 @@ func testAcceptance(t *testing.T, when spec.G, it spec.S) {
 			configuredRunImage := "some-registry.com/pack-test/run1"
 			cmd := packCmd("set-run-image-mirrors", "pack-test/run", "--mirror", configuredRunImage)
 			output := h.Run(t, cmd)
-			h.AssertEq(t, output, "Run Image 'pack-test/run' configured with mirror 'some-registry.com/pack-test/run1'\n")
+			h.AssertContains(t, output, "Run Image 'pack-test/run' configured with mirror 'some-registry.com/pack-test/run1'\n")
```

Comment:
Should be able to go back to equality when prefixes are removed for the CLI right?